framework/auth/campaigns.py

```diff
@@ -0,0 +1,39 @@
+import httplib as http
+
+from werkzeug.datastructures import ImmutableDict
+from framework.exceptions import HTTPError
+
+from website import mails
+from website.util import web_url_for
+
+CAMPAIGNS = ImmutableDict({
+    'prereg': {
+        'system_tag': 'prereg_challenge_campaign',
```

Comment:
Thanks for making the system tag less ambiguous.

But building on that- do we really need a special `system_tag` key (and a function to convert old->new name) at all?

It seems that `campaign` is passed in from the front end, in which case this function could be removed if the front-end (and unit tests) simply passed in the correct name originally. I'm in favor of removing unnecessary aliasing code where possible; it reduces boilerplate and makes following the flow of the program easier.

See front-end code here:
https://github.com/CenterForOpenScience/osf.io/pull/4632/files#diff-96b2a4b9793e1418b2ff319410c1be9dR24

This may also affect other areas like URLs; not sure. Thoughts?